android/guava/src/com/google/common/collect/CompactHashMap.java

```diff
@@ -194,6 +222,36 @@ int allocArrays() {
     return expectedSize;
   }
 
+  @SuppressWarnings("unchecked")
+  @VisibleForTesting
+  @NullableDecl
+  Map<K, V> delegateOrNull() {
+    if (table instanceof Map) {
+      return (Map<K, V>) table;
+    }
+    return null;
+  }
+
+  Map<K, V> createHashFloodingResistantDelegate(int tableSize) {
+    return new LinkedHashMap<>(tableSize, 1.0f);
```

Comment:
Should this be a HashMap according to the comment in line 114 and the override in CompactLinkedHashMap?

Reply:
Thanks. The other one should be `LinkedHashMap`, too. I'll fix it.